suites/nautilus/live/sanity_container.yaml

```diff
@@ -0,0 +1,106 @@
+tests:
```

Comment:
Can we list out out steps in test suite as defined in the this suite, https://github.com/red-hat-storage/cephci/blob/master/suites/pacific/cephadm/tier1_cephadm_upgrade.yaml#L1-L28

Reply:
please add steps w.r.t setting up environment for ceph live testing.

Reply:
@sunilkumarn417 Thanks for the hint, sure will add those steps in the suite

Reply:
@sunilkumarn417 It's done, Please review the changes.